### ISSUE TITLE
better Relationship constructor?

### ISSUE THREAD

**Issue by zvr** [spdx/tools-python#336]
A `Relationship` has a _type_ and connects a _from_ element and a _to_ element.

For example, it might represent that an SPDX Document describes a Package.

There should be a more Pythonic way to construct them than the current "creating a string" approach:

    Relationship(doc.spdx_id + " " + RelationshipType.DESCRIBES.name + " " + pkg.spdx_id)

**Comment by armintaenzertng** [comment]
Yes, we will have a new `Relationship` dataclass in the upcoming refactored data model, see [here](https://github.com/spdx/tools-python/blob/refactor-python-tools/src/model/relationship.py).
